.github/workflows/generate_issues.yml

```diff
@@ -8,6 +8,8 @@ on:
   schedule:
     # Run every day at 4am (PST) - cron uses UTC times
     - cron:  '0 12 * * *'
+permissions:
+  issues: write
```

Comment:
LGTM after fixing formatting.

Reply:
Done. Keeping it at the workflow level to apply to both jobs